src/ert/config/queue_config.py

```diff
@@ -147,9 +147,30 @@
 }
 
 
+@dataclass
+class QueueMemoryStringFormat:
+    suffixes: List[str]
+
+    def validate(self, mem_str_format: str) -> bool:
+        return (
+            re.match(
+                r"\d+(" + "|".join(self.suffixes) + ")$",
+                mem_str_format,
+            )
+            is not None
+        )
+
+
+queue_memory_usage_formats: Mapping[str, QueueMemoryStringFormat] = {
+    "SLURM": QueueMemoryStringFormat(suffixes=["", "K", "M", "G", "T"]),
+    "TORQUE": QueueMemoryStringFormat(suffixes=["kb", "mb", "gb", "KB", "MB", "GB"]),
+}
```

Comment:
Are the queue system formats documented somewhere in the ert docs, or should the user check with the queue system docs directly?

Reply:
The existing Torque and Slurm docs for Ert mention units sufficiently yes. At least sufficiently for deprecated parameters. 

The new docs for REALIZATION_MEMORY uses `Gb` in its example. I think that is enough for any relevant usage of the keyword.